front/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { todosAPI } from '../../api/todos-api';
+import { TodosArr } from '../../types';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootStateType } from '../../redux/store';
+import { setTodos } from '../../redux/todo-reducer';
+import { setIsFetching } from '../../redux/app-reducer';
+import { AxiosError } from 'axios';
+import { addMessage } from '../../redux/messages-reducer';
+import styles from './Home.module.scss';
+
+type useQueryType = {
+  error: AxiosError | null
+  [propName: string]: any
+}
+
+const Home = (): JSX.Element => {
+  const dispatch = useDispatch();
+  const { todos } = useSelector((state: RootStateType) => state.todos)
+
+  const {
```

Comment:
I guess you re-save data from React-Query into Redux just for learning purposes. Instead of such a synthetic move, may we have i18n messages in Redux and keep data in React-Query's cache exclusively?